proto/near/core.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+import "near/get_public_key.proto";
+
+import "error.proto";
+
+package near;
+
+message Query {
+  oneof request {
+    GetPublicKeyRequest get_public_keys = 1;
+    GetPublicKeyRequest get_public_key = 2;
```

Comment:
Since we will enforce the user to verify the generated public key, It'll be better if we can name it accordingly.

```suggestion
    GetPublicKeyRequest get_user_verified_public_key = 2;
```

Reply:
Updated, pls re-review